pkg/config/config.go

```diff
@@ -1088,6 +1088,8 @@ var (
 // NewConfig creates a new config instance with default value.
 func NewConfig() *Config {
 	conf := defaultConf
+	// TODO(hawkingrei): it is an experimental feature, we will close it before the next release.
```

Comment:
so the opening here is for what？

Reply:
Because the config is from the client-go. In the client-go, we set the default value as OFF. For the test, we have to set the value as ON here.